corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -2,21 +2,18 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 import hashlib
+import mock
```

Comment:
nit: Is `mock` in stdlib? I think technically it belongs in the next section of "third-party" library imports (where it was before).

Reply:
that's true - I was confusing it with ```unittest.mock```, which is part of py3's stdlib